docs/guides/dapis/subscribing-to-dapis/index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Subscribing to dAPIs
+sidebarHeader: Guides
+sidebarSubHeader:
+pageHeader: Guides → dAPIs
+path: /guides/dapis/subscribing-to-dapis/index.html
+outline: deep
+tags:
+---
+
+<PageHeader/>
+
+<SearchHighlight/>
+
+<FlexStartTag/>
+
+# {{$frontmatter.title}}
+
+> <Video src="https://www.youtube.com/embed/Wq8tLxbhJBQ"/>
+
+[dAPIs](/explore/dapis/using-dapis.md#managed-dapis) are datafeeds that are
+sourced directly from multiple
+[first-party oracles](/explore/airnode/why-first-party-oracles) and aggregated
+using Signed data. It utilizes first-party oracles, enabling data to be served
+directly from reputable API providers. This guide will walk you through the
+process of subscribing to a dAPI.
+
+## API3 Market
+
+The [API3 Market](https://market.api3.org) provides all the tooling to
+seamlessly access dAPI services. This end-to-end process consists of:
+
+- [Exploring, selecting and configuring your dAPI](/guides/dapis/subscribing-to-dapis/#exploring-selecting-and-configuring-your-dapi)
+  <br/>
+- [Activating your dAPI](/guides/dapis/subscribing-to-dapis/#activating-your-dapi)
+  <br/>
+- [Setting up OEV and getting the proxy address](/guides/dapis/subscribing-to-dapis/#setting-up-oev-and-getting-the-proxy-address)
+  <br/>
+
+## Exploring, selecting and configuring your dAPI
+
+The [API3 Market](https://market.api3.org/) provides a list of all the dAPIs
+available across multiple chains including testnets. You can filter the list by
+mainnet or testnet chains. After selecting the chain, you can now search for a
+specific dAPI by name. Once selected, you will land on the details page
+[(eg ETH/USD on Ethereum Sepolia)](https://market.api3.org/dapis/ethereum-sepolia-testnet/ETH-USD)
+where you can find more information about the dAPI.
+
+<img src="../../assets/images/dapi-3.png" style="width:100%"/>
+
+If a dAPI is already activated, it will provide information like the current
+aggregated on-chain value, funding address, current configuration, and the
+expiration date.
+
+Once you have selected your dAPI, you will then be presented with an option to
+purchase and activate it.
```

Comment:
This sentence reads a bit out of order given the sentence above it starts with "If a dAPI is already activated..."